config/identitystore/config.go

```diff
@@ -0,0 +1,14 @@
+package identitystore
+
+import (
+	"github.com/crossplane/upjet/pkg/config"
+)
+
+// Configure adds configurations for the identitystore group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_identitystore_group", func(r *config.Resource) {
+		// Display name is required by terraform, and while it's not part of the external name or terraform id, it is
+		// how the group is displayed, and it's immutable.
+		r.ExternalName.IdentifierFields = append(r.ExternalName.IdentifierFields, "display_name")
```

Comment:
Is there an easy, idiomatic golang way to append this only if it's not already present? In e.g. python I'd probably do it by converting to a set and then back to a list. Or maybe it's not worth worrying about since this happens at generation time.

Or maybe it's better to leave this off? I added it in an effort to get the field marked as required in the generated schema, but subsequently I realized that many fields which are actually required by terraform are not marked as required in the schema, but are instead validated at the kubernetes level using slightly more sophisticated logic that takes into account `initProvider` and `managementPolicies`. 

Conceptually it makes sense that the display name would be an identifier field. But it's not necessary to mark it as such in order for the managed resource to work properly.